### DISCUSSION TITLE
Range

### DISCUSSION THREAD

**Discussion by jeffreylauwers** [nl-design-system/backlog#231] [Component suggestions]
<img width="480" alt="visueel voorbeeld van het range component" src="https://github.com/nl-design-system/backlog/assets/545207/09c9587a-addf-4348-9e2b-253c34f46e81">

We willen graag meer leren over het Range component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook of screenshots. Met deze informatie kunnen we bewijzen dat het component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

[Link naar bericht in Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1711019864302469)

---

## Naam
Range

## Doel
Invoerveld voor een getal dat je bij benadering kunt kiezen door te schuiven tussen een minimum en maximum.

## Ook bekend als
Slider, Range Slider

## Gerelateerde componenten
[Number Input](https://github.com/orgs/nl-design-system/discussions/235)

## Onderzoek
[Slider Design: Rules of Thumb - Nielsen Norman Group](https://www.nngroup.com/articles/gui-slider-controls/)
Artikel over het ontwerp van Range Sliders.

[Balancing Exploration and Precision with Sliders - Nielsen Norman Group](https://www.nngroup.com/articles/sliders-knobs/)
Artikel over het gebruik van Range Sliders.

[Overly incremented sliders: "Fun" tools that become obstacles - Nielsen Norman Group](https://www.nngroup.com/articles/navigation-cognitive-strain/#toc-4-overly-incremented-sliders-fun-tools-that-become-obstacles-7)
Inzicht uit onderzoek waar een Range Slider werd toegepast.

[5 Requirements for Slider Interfaces - Baymard Institute](https://baymard.com/blog/slider-interfaces)
Artikel met tips voor Range Sliders gebaseerd op onderzoek.

[Sliders degrade UX (so do this instead) - Adam Silver](https://adamsilver.io/blog/sliders-degrade-ux-so-do-this-instead/)
Artikel waarin wordt uitgelegd waarom slides problemen kunnen geven, en welke form controls je beter kunt inzetten.

**Comment by jeffreylauwers** [comment]
## Amsterdam

- [Figma](https://www.figma.com/file/9IGm6IdPUYizBNGsUnueBd/Amsterdam-Design-System?type=design&node-id=5562-45662&mode=design&t=3RM9VrRKprFqj2Te-4)
- 'Live' te bekijken op [onderzoek.amsterdam.nl](https://onderzoek.amsterdam.nl/zoek)

---

### Screenshots / Varianten

**Figma**

Default
![Scherm­afbeelding 2024-03-21 om 12 51 09](https://github.com/nl-design-system/backlog/assets/545207/036c6231-2094-43a1-8b49-ead3ac27ba26)

---

### Notities
- Onder de naam: Slider.
- Dubbele thumbs.
- Optie voor input functionaliteit.
- Inputfield onder Range.
- Thumb is rond en heeft dezelfde blauwe kleur als de fill en track. Thumb heeft een witte border.

**Comment by jeffreylauwers** [comment]
## DUO

[Documentatie website](https://uno.dfront.rijkscloud.nl/#/components/slider)

---

### Screenshots / Varianten

**Documentatie website**
![Scherm­afbeelding 2024-03-21 om 13 13 59](https://github.com/nl-design-system/backlog/assets/545207/fb099491-8ca9-4a6d-9b93-ad0f24ea1ab3)

---

### Notities
- Onder de naam: Slider.
- Vaste input functionaliteit.
- Inputfield links van Range.
- Tijdens het schuiven wordt er boven de thumb een tooltip met waarde getoond.
- Thumb is rond en heeft dezelfde kleur als de fill.
- Fill is blauw.
- Track is lichtgrijs.

**Comment by jeffreylauwers** [comment]
## Rotterdam

### Niet meer van toepassing!
Rotterdam gaat dit component niet meer aanbieden.

---

### Screenshots / Varianten

**Figma**

Default
![Scherm­afbeelding 2024-03-21 om 13 07 46](https://github.com/nl-design-system/backlog/assets/545207/a7a744fd-65c1-4605-b319-2d00e57aa4bf)

Range slider
![Scherm­afbeelding 2024-03-21 om 13 07 51](https://github.com/nl-design-system/backlog/assets/545207/815a6434-611e-4125-bdba-6dad0493cf03)


---

### Notities
- Onder de naam: Sliders.
- Default is enkele thumb.
- Bij default mogelijkheid om waarde rechts van track te tonen.
- Range is 2 thumbs.
- Bij range worden waarden onder de thumb getoond.
- Thumb is vierkant en heeft 'grip' visualisatie.
- Fill is groen.
- Variant voor een disabled state.

**Comment by Yolijn** [comment]
## ✨ Dit component is nu Help Wanted ✨


Help je mee hem door de Community stappen te krijgen?
[Range op het Community bord](https://github.com/orgs/nl-design-system/projects/29/views/1?filterQuery=%2336)
